ios/Peepz/Peepz/Authentication/LoginView.swift

```diff
@@ -0,0 +1,42 @@
+import SwiftUI
+import GoogleSignIn
+
+struct LoginView: View {
+    @EnvironmentObject var model: PeepzModel
+
+    var body: some View {
+        VStack {
+            Image(systemName: "person.circle").imageScale(.large)
+            GoogleSignInViewController()
+
+            // isActive - means is it showing its destination
+            NavigationLink(destination: GalleryView(), isActive: $model.isAuthenticated) { EmptyView() }
```

Comment:
This is prety cool it basically means if this will turn to false the view is automatically being dismissed.